platform-core/src/main/java/com/webank/wecube/platform/core/controller/PluginPackageController.java

```diff
@@ -33,6 +43,13 @@
     @Autowired
     private PluginPackageValidator validator;
 
+    @GetMapping("/my_menus")
```

Comment:
please use hyphen(-) instead of underscore(_)

Reply:
The naming issue has fixed